addons/web/static/src/js/widgets/colorpicker.js

```diff
@@ -287,6 +287,10 @@ var ColorpickerWidget = Widget.extend({
     _onClick: function (ev) {
         ev.originalEvent.__isColorpickerClick = true;
         $(ev.target).find('> .o_opacity_pointer, > .o_slider_pointer, > .o_picker_pointer').addBack('.o_opacity_pointer, .o_slider_pointer, .o_picker_pointer').focus();
+        if (ev.target.dataset.colorMethod === 'hex') {
+            const {value} = ev.target;
+            ev.target.setSelectionRange(1, value.length);
```

Comment:
It was to avoid selecting the first character `#`, that you usually not select when copying an hexadecimal value, but it's also possible to select the whole input include the `#` !

Reply:
@sbuhl Opinion? What do you want? ;) Personally the only thing I use to copy color is the "pipette" picker from Firefox and it copies the `#` which seems normal to me... Have you an example of software where we can actually copy a hex color without the `#` @younn-o?

Reply:
Hello @qsm-odoo 
I think we'll find a lot of contradictory things.
e.g. on https://color.adobe.com/create/color-wheel you double click on the code and it only copy the hexa, without the #. But on Chrome and Firefox you copy everything. 
Maybe it's better to do the same than the browsers then. 
May I ask you to change @younn-o  ?

Reply:
Ok, I am changing that :)